js/common/util.go

```diff
@@ -32,17 +35,17 @@ func RunString(rt *goja.Runtime, src string) (goja.Value, error) {
 }
 
 // RunES6String Runs an ES6 string in the given runtime. Use this rather than writing ES5 in tests.
-func RunES6String(rt *goja.Runtime, src string) (goja.Value, error) {
+func RunES6String(tb testing.TB, rt *goja.Runtime, src string) (goja.Value, error) {
```

Comment:
Ah, we should move `RunES6String()` to `testutils` or some sub-package there... This is going to cause us to compile the whole Go `testing` in our release builds.

Reply:
fixed in 251a88d4